mysite/polls/views.py

```diff
@@ -0,0 +1,43 @@
+from django.shortcuts import render
+import search
+import json
+from django.http import JsonResponse
+from django.http import HttpResponse
+import re
+# Create your views here.
+
+
+def make_link_for_html(movie_id):
+    link = '<a href = "http://localhost:8000/polls/info/?q=' \
+            + movie_id + '"> Link </a>'
+    return link
+
+
+def make_link(movie_id):
+    link = 'http://localhost:8000/polls/info/?q=' \
```

Comment:
Better to use string formating than concatenation: http://www.pythonforbeginners.com/concatenation/string-concatenation-and-formatting-in-python